client/rpcserver/handlers.go

```diff
@@ -862,8 +862,13 @@ Registration is complete after the fee transaction has been confirmed.`,
     quote (int): The BIP-44 coin index for the market's quote asset.
     qty (int): The number of units to buy/sell. Must be a multiple of the lot size.
     rate (int): The atoms quote asset to pay/accept per unit base asset. e.g.
-      156000 satoshi/DCR for the DCR(base)_BTC(quote).
-    immediate (bool): Require immediate match. Do not book the order.`,
+      56000 satoshi/DCR for the DCR(base)_BTC(quote).
```

Comment:
Just noticing the unintentional change from 156000 => 56000.   When this was written (May 28, 2020), it was describing an exchange rate of 0.00156 BTC/DCR, so I'd not object to a new optimistic example rate of 543210 (0.0054321) instead, but definitely not 0.00056 :)